docs/workforce-management-framework/topics/con-datasync-cloud-module.adoc

```diff
@@ -0,0 +1,42 @@
+= {WFM-RC-NameLong} Datasync Module
+
+As with all synchronisation tasks Datasync Module has cloud and client sides. This is a implementation of 
+the Feedhenry Sync Framework for advanced usage see the 
+link:{WFM-RC-SyncFrameworkURL}{WFM-RC-Branch}/docs[sync framework documentation]. 
+The primary funtion of Datasync is to synchronise mobile data between mobile and a backend server.
```

Comment:
Suggestion:
"The primary function the *Datasync* module is to synchronise data between the client (mobile) and a backend server (cloud)." - 'cloud' may be wrong in this instance.

Reply:
"Client Apps" and "Cloud Apps" mentioned below so maybe "a Client App, and a Cloud App running on a backend server"

What very important here is that whatever the decision is, it's reflected throughout this PR so there is consistency.

Reply:
````The primary funtion of Datasync module is to synchronise data between mobile(Client App) and a backend server(Cloud App).````